File: native/packages/react-native-backpack/libraries/Panel/android/react/src/main/java/net/skyscaner/backpack/panel/PanelWithDivider.java
package net.skyscaner.backpack.panel;

import com.facebook.react.uimanager.SimpleViewManager;
import com.facebook.react.uimanager.ThemedReactContext;

import net.skyscanner.backpack.panel.BpkPanelWithDivider;

public class PanelWithDivider extends SimpleViewManager<BpkPanelWithDivider> {

    public static final String REACT_CLASS = "BPKPanel";

    @Override
    public String getName() {
        return REACT_CLASS;
    }

    @Override
    protected BpkPanelWithDivider createViewInstance(ThemedReactContext reactContext) {
        return new BpkPanelWithDivider(reactContext);
    }
}

File: native/packages/react-native-backpack/libraries/Panel/android/react/src/main/java/net/skyscaner/backpack/panel/PanelWithDivider.kt
package net.skyscaner.backpack.panel

import com.facebook.react.uimanager.SimpleViewManager
import com.facebook.react.uimanager.ThemedReactContext

import net.skyscanner.backpack.panel.BpkPanelWithDivider

class PanelWithDivider : SimpleViewManager<BpkPanelWithDivider>() {

    override fun getName(): String {
        return REACT_CLASS
    }

    override fun createViewInstance(reactContext: ThemedReactContext): BpkPanelWithDivider {
        return BpkPanelWithDivider(reactContext)
    }

    companion object {

        val REACT_CLASS = "BPKPanel"
    }
}

